hat/assets/js/apps/Iaso/domains/projects/components/CreateEditProjectDialog.tsx

```diff
@@ -211,21 +212,34 @@ const ProjectsDialog: FunctionComponent<Props> = ({
             !isFetchingFeatureFlags,
         [project, isFetchingFeatureFlags],
     );
+    const titleMessage =
+        dialogType === 'create'
+            ? formatMessage(MESSAGES.create)
+            : formatMessage(MESSAGES.updateProject);
 
     return (
-        // @ts-ignore
-        <ConfirmCancelDialogComponent
+        <ConfirmCancelModal
             allowConfirm={allowConfirm}
             titleMessage={titleMessage}
-            onConfirm={closeDialog => onConfirm(closeDialog)}
+            onConfirm={onConfirm}
+            onCancel={() => {
+                setProject(initialProject(null));
+                setTab('infos');
+                closeDialog();
+            }}
+            maxWidth={
+                tab === 'feature_flags' &&
+                project.feature_flags.value.length > 0
+                    ? 'lg'
+                    : 'xs'
+            }
```

Comment:
This is creating a visual bug while adding or removing FF:

https://github.com/BLSQ/iaso/assets/12494624/9e86ac54-e4a3-46aa-b1fb-94bf6ef8205a

So we have two solutions here:

1) The easiest one: having a container on FF tab setting the width a bit larger (lg is too big). It will change Dialog widht while changing tab
2) The hard one: as @quang-le said, maybe use a table + checkboxes like the permissions ?